scripts/update_hyphenation.sh

```
@@ -25,3 +25,4 @@ process it
 process uk
 process pl
 process sv
+process fi
```

Shellcheck (0.11.0)

[warning] Use semicolon or linefeed before 'fi' (or quote to make it literal).

(SC1010)